StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -392,7 +392,7 @@ extension NativeFlowController: PartnerAuthViewControllerDelegate {
             nativeFlowController: self,
             dataManager: dataManager
         )
-        pushViewController(accountPickerViewController, animated: true)
+        pushViewController(accountPickerViewController, animated: false)
```

Comment:
Are we sure we always want to not animate? `didCompleteWithAuthSession` might be called from multiple places (rather than just after auth call), here is one:

https://github.com/stripe/stripe-ios/blob/c4b836b2/StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthViewController.swift#L304-L305